### ISSUE TITLE
Modify success message to make it more generic

### ISSUE THREAD

**Issue by tovarnakM** [NomicFoundation/hardhat#3960]
https://github.com/NomicFoundation/hardhat/blob/de94479cc80bdb25de864ea1df366b0d6f3057cc/packages/hardhat-verify/src/index.ts#L524

Hi we are using hardhat plugin for our custom blockchain solution and explorer. We know that you support Etherscan based explorers but surely not only those.

We are wondering whether it is possible to make success compilation message above more generic - e.g. on the block explorer. You are already using more generic messaging couple lines above ...Successfully submitted source code for contract
...for verification on the block explorer. Waiting for verification result...

Thanks for reply

**Comment by fvictorio** [comment]
Makes sense. We should either drop the "on Etherscan", or maybe show the base URL.
